NO_LAVH_001 (#362)
NINAnor/ecRxiv: R/NO_LAVH_001.qmd

    "Indicator ID",
    "Indicator Name",
    "Continent",
    "Country",
    "Ecosystem Condition Typology Class", 
    "Realm",
    "Biome",
    "Ecosystem",
    "Year added", 
    "Last update",
    "Version",
    "Version comment",
    "Normalised",
    "Spatial aggregation pathway")) |>   
  kable()

```

:::

::: {.callout-tip collapse="true"}

## Log

<!--# Update this logg with short messages for each update -->
- May 28, 2026 - Started work
:::

<hr />

<!--# Document you work below.  -->

```{r fig-leming}
#| fig-cap: 'Reindeer feeding on lichens. Photo: Olav Strand.'
knitr::include_graphics(here::here("img/lav_olav_strand.jpg"))
```

## 1. Summary

<!--# 
With a maximum of 300 words, describe the indicator in general terms as you would to a non-expert. Think of this as a kind of common language summary. Follow the four numbers sub-headers below. under point 2, it is possible under  include a bullet point list of the specific steps in the workflow. 
 -->

1. **Background and rationale**:
<!-- What does the metric represent? Why is this relevant for describing ecosystem condition in this ecosystem?
What are the main anthropogenic impact factors? -->

> Lichens, and lichen heaths, are common and important features of alpine ecosystems. They are important food sources for reindeer, hence the common Norwegian term 'reindeer lichens'. This group of lichens are characteristically white in colour. This makes them important in climate forcing by increasing the surface albedo. It also makes them easily recognisaible via remote sensing. 

Reviewer: ```suggestion
Lichens, and lichen heaths, are common and important features of alpine ecosystems. They are important food sources for reindeer, hence the common Norwegian term 'reindeer lichens'. This group of lichens are characteristically white in colour. This makes them important in climate forcing by increasing the surface albedo (reflectivness) . It also makes them easily recognisable via remote sensing. 
```

>  The cover and biomass of lichen is affected by several anthopogenic drivers, including climate change and land use change. Climate warming and reduced livestock grazing causes shubbification and forest advance, whereas lichen cover can also become reduced because of reindeer overgrazing.   

Reviewer: ```suggestion
 The cover and biomass of lichen is affected by several anthopogenic drivers, including climate change and land use change. Climate warming and reduced livestock grazing causes shubbification (an increase in the abundance and density of shrub species) and forest advance, whereas lichen cover can also become reduced because of reindeer overgrazing.   
```

>  The cover and biomass of lichen is affected by several anthopogenic drivers, including climate change and land use change. Climate warming and reduced livestock grazing causes shubbification and forest advance, whereas lichen cover can also become reduced because of reindeer overgrazing.   

Author: ```suggestion
 The cover and biomass of lichen is affected by several anthopogenic drivers, including climate change and land use change. Climate warming and reduced livestock grazing causes shrubification and forest advance, whereas lichen cover can also become reduced because of reindeer overgrazing.   
```

2. **Methods**:
<!-- What kind of data is used? How is the data customized (modified, estimated, integrated) to fit its purpose as an indicator? -->

> Lichen biomass is estimated by using a machine learning model on landsat satellite imagery, every year from 19884 to 2024, for wild reindeer areas. Areas dominated by semi-domesticated raindeer er not included. The variable is defined as the proportion of area with lichen heath in good condition, following the 'Kvalitetsnorm for villrein'.  

Reviewer: ```suggestion
Lichen biomass is estimated by using a machine learning model on landsat satellite imagery, every year from 19884 to 2024, for wild reindeer areas. Areas dominated by semi-domesticated raindeer are not included. The variable is defined as the proportion of area with lichen heath in good condition, following the 'Kvalitetsnorm for villrein'.  
```

We get the data from the Norwegian Nature Index database [@nater_naturindeks_2025] where also the X~100~ reference values are stored.

3. **Interpretation**:
<!-- Add a very short description of how the indicator values should be interpreted. Here's an example for a bumblebee indicator: An indicator value of 1 signifies an intact bumblebee community. Values decreasing toward zero reflect negative ecological change, specifically the decline of species expected to be present. This metric does not account for potentially positive changes, such as increases in species' abundance or the arrival of new species. Tip: Add text on the line directly following the subheader, and don't introduce blank lines. This way the whole paragraph renders as a block quote. -->

> An indicator value of 1 means that the are area with lichen heath in good condition (i.e. with sufficient lichen biomass) is equal to or greater than that estimated under the reference condition. And indicator value decreasing towards zero indicates a reduction in the ratio, intil with an indicator value of zeroe, where all area are in poor condition. 

Reviewer: ```suggestion
An indicator value of 1 means that the are area with lichen heath in good condition (i.e. with sufficient lichen biomass) is equal to or greater than that estimated under the reference condition. An indicator value decreasing towards zero indicates a reduction in condition relative to the reference and an indicator value of zero indicating all areas are in poor condition. 
```

Reviewer: Not sure if I worded thie correctly

> An indicator value of 1 means that the are area with lichen heath in good condition (i.e. with sufficient lichen biomass) is equal to or greater than that estimated under the reference condition. And indicator value decreasing towards zero indicates a reduction in the ratio, intil with an indicator value of zeroe, where all area are in poor condition. 

Author: ```suggestion
An indicator value of 1 means that the area with lichen heath in good condition (i.e. with sufficient lichen biomass) is equal to or greater than that estimated under the reference condition. And indicator value decreasing towards zero indicates a reduction in the ratio, intil with an indicator value of zeroe, where all area are in poor condition. 
```

4. **Results and development needs**:
<!-- Briefly, what does the indicator say about the ecosystem condition. What is the current status of  the metric (can it be used or is it still in development), and what immediate tasks remain to improve the indicator or to make the indicator operational -->

> The indicator value for Norway as a whole in 2024 is 0.73 [95% CI = 0.72-0.74], something which can be interpreted as *good condition*. The national indicator values are showing a weak decline since 2010. The indicator has highest values in southern Norway (0.89) and lowest values in west (0.66). Development needs include to calculate this same indicator for all areas, including municipalities dominated by semi-natural reindeer. 

Reviewer: ```suggestion
The indicator value for Norway as a whole in 2024 was 0.73 [95% CI = 0.72-0.74], something which can be interpreted as *good condition*. The national indicator values are showing a weak decline since 2010. The indicator has highest values in southern Norway (0.89) and lowest values in west (0.66). Development needs include to calculate this same indicator for all areas, including municipalities dominated by semi-natural reindeer. 
```

B2 - Structural state characteristics

<!-- 
### 3.1.1 Other condition typologies -->

<!--# Optional: Add text about other standards or typologies, such as alternative national condition typologies, and how the indicator relates to these -->

### 3.2 Ecosystem condition characteristic

<!--# 

Describe the ecosystem condition characteristic represented in the indicator. See 10.3897/oneeco.6.e58218 for information on what these characteristics might be.
For example, and indicator called 'Trenching in mires' could be made to represent an ecosystem characteristic 'Intact hydrology'. The term 'characteristic' is used similar to the term 'criteria' in Multiple Criteria Decision Making.  

-->

The ecosystem characteristic represented in this indicator is an ecosystem where the vegetation, in terms of cover, composition and biomass, is close to that we expect under the reference condition. 

### 3.3 Collinearities with other indicators

<!--# Describe known collinearities with other metrics (indicators or variables) that could become problematic if they were also included in the same Ecosystem Condition Assessment as the indicator described here. -->

> Non that are very obvious. The indicator could be correlated to indicators on reindeer grazing pressure.

Reviewer: ```suggestion
None that are very obvious. The indicator could be correlated to indicators on reindeer grazing pressure.
```

### 3.4 Impact factors

<!--# Describe the main natural and anthropogenic factors that affecst the metric -->
The biomass and cover of lichen heaths is reduced by overgrazing and shrub/forest enchroachment due to changes in land use and land use intensity.

### 3.5 Spatial aggregation

<!-- Describe in word the same aggregation pathway presented in the top yaml (the metadata). If you have scaled against the reference value, it is especially important to know if you did any avergaing or summations etc. beforehand. An example text for a rather simple case could be: "The variable is scaled at the initial polygon level and the point estimate is a area-weighted arithmetic mean of all polygins. There is no transformation, and truncation is not needed as the indicator becomes naturally bound between 0 and 1".  -->

## 4. Reference condition and levels

### 4.1 Reference condition

<!--# Define the reference condition (or refer to where it is defined). Note the distinction between reference condition and reference levels 10.3897/oneeco.5.e58216  -->

> The reference condition is a moutain ecosystem with minimal human impact and a climate similar to the period 1961-1990 [@nybo_fagsystem_2017].

Reviewer: ```suggestion
The reference condition is a mountain ecosystem with minimal human impact and a climate similar to the period 1961-1990 [@nybo_fagsystem_2017].
```

#### 4.2.1 Spatial resolution and validity

<!--# 

Describe the spatial resolution of the reference levels. E.g. is it defined as a fixed value for all areas, or does it vary. Also, at what spatial scale are the reference levels valid? For example, if the reference levels have a regional resolution (varies between regions), it might mean that it is only valid and correct to use for normalising local variable values that are first aggregated to regional scale. However, sometimes the reference levels are insensitive to this and can be used to scale variables at the local (e.g. plot) scale. 

 -->
 The reference values vary between municipalities in Norway.

## 5. Uncertainties
### 5.1 General uncertainties
<!--# Describe the main reasons why you might not trust 100% the final value to represent the given ecosystem characteristic for the give ecosystem type. What aspects of this condition metric are most sensitive to subjective decitions? Is the spatial sampling sufficient? Does the choice of modeling framework affect the values? Is the metric relevant do describe ecosystem condition for the give ecosystem type?  -->

> The main source of uncertainty arises from the spatial bias, i.ee that municipalities dominated by semi-domestic reindeer are excluded. 

Reviewer: ```suggestion
The main source of uncertainty arises from the spatial bias, i.e that municipalities dominated by semi-domestic reindeer are excluded. 
```

The reported uncertainties reflect the uncertainties around the true parameter means, i.e. they are inferrential uncertainties for the true value of the indicator for the given area and time combination.  

## 6. References

<!--# You can add references manually or use a citation manager and add intext citations as with crossreferencing and hyperlinks. See https://quarto.org/docs/authoring/footnotes-and-citations.html -->

::: {#refs}
:::

## 7. Import and prepare datasets

<!--# All the data import and preparation should be done in this chapter, before the analysis chapter. As a minimum, add code for importing data, and describe or show the data structure. You may also explain certain variables, what they represent, and their distributions -->

### 7.1 Naturindeks

<!--# Import and prepare the first dataset for analysis. Change the header from Dataset A to the name of the actual dataset. -->

We download the dataset from the Naturindeks database using the API. 

> The function giver read-only access without requiring a username and password.

Reviewer: ```suggestion
The function gives read-only access without requiring a username and password.
```
